docs/source/developer/fields.md

```diff
@@ -394,25 +394,6 @@ Clear:
 }
 ```
 
-Multi:
-
-Group multiple operations into one
-
-```json
-{
-    "field": {
-        "op": "multi",
-        "value": [{
-            "op": "assign",
-            "value": {
-                "key": "foo",
-                "value": "bar"
-            }
-        }]
-    }
-}
-```
-
```

Comment:
Why this part is deleted?

Reply:
Seems that BucketListField & BucketDictField doesn't support "multi" operations, maybe this is a bug?
 doesn't seem to support xx operations, maybe this is a bug?
`{
    "reason": "deserializationError",
    "details": "Could not deserialize the content",
    "type": "DeserializationError",
    "eid": "630045b5ef734b369e93dd726c8f75d4",
    "message": "DeserializationError()",
    "deserialization_errors": [
        {
            "message": "\"multi\" not a valid operation",
            "field": "field"
        }
    ]
}`

Reply:
Sorry, yeah, I thought this was the part of PatchFields